doc/dash/dash-sonic-hld.md

```diff
@@ -723,83 +725,132 @@ For single endpoint, implmentation shall simply create a sai_dash_tunnel object
-|                       | vnet         |                 | sai_outbound_ca_to_pa_entry_t.dest_vni          | VNET's VNI                                    |
-|                       | ip_address   |                 | sai_outbound_ca_to_pa_entry_t.dip               |                                               |
-|                       |              | routing_type    |                                                 |                                               |
-|                       |              | underlay_ip     | SAI_OUTBOUND_CA_TO_PA_ENTRY_ATTR_UNDERLAY_DIP   |                                               |
-|                       |              | mac_address     | SAI_OUTBOUND_CA_TO_PA_ENTRY_ATTR_OVERLAY_DMAC   |                                               |
-|                       |              | metering_bucket | SAI_OUTBOUND_CA_TO_PA_ENTRY_ATTR_COUNTER_ID     |                                               |
-|                       | vnet*        |                 | sai_pa_validation_entry_t.vnet_id               | VNET's VNI                                    |
-|                       |              | underlay_ip*    | sai_pa_validation_entry_t.sip                   | SAI_PA_VALIDATION_ENTRY_ATTR_ACTION is permit |
-| DASH_ROUTE_RULE_TABLE |              |                 |                                                 |                                               |
-|                       | eni          |                 | sai_inbound_routing_entry_t.eni_id              |                                               |
-|                       | vni          |                 | sai_inbound_routing_entry_t.vni                 |                                               |
-|                       | prefix       |                 | sai_inbound_routing_entry_t.prefix              |                                               |
-|                       |              | action_type     |                                                 |                                               |
-|                       |              | priority        | sai_inbound_routing_entry_t.priority            |                                               |
-|                       |              | protocol        |                                                 |                                               |
-|                       |              | vnet            | SAI_INBOUND_ROUTING_ENTRY_ATTR_SRC_VNET_ID      |                                               |
-|                       |              | pa_validation   | SAI_INBOUND_ROUTING_ENTRY_ATTR_ACTION           | use PA_VALIDATE if true                       |
-|                       |              | metering_bucket |                                                 |                                               |
+### 3.2.15 DASH outbound port map table
+```
+DASH_OUTBOUND_PORT_MAP_TABLE:{{map_id}}
```

Comment:
@theasianpianist this is 2 level hierarchy - there can be multiple port_map table instances. And each port_map table instance can have multiple port ranges within it each with its own backend_ip.

So the port_range needs to be added to the key. 
SAI API defines this as 2 objects: outbound-port-map and outbound-port-map-port-range.
We could also model this as 2 tables in APP DB - say PORT_MAP and PORT_MAP_RANGE table
  The PORT_MAP table entries will be keyed by port_map_id.  
  The PORT_MAP_RANGE table entries will be keyed by {port_map_id , port_range}.
The CA-PA mapping points to the PORT_MAP table instance.

Reply:
On second thought, we already have a similar pattern with ROUTE_GROUP and ROUTE_TABLE, I'll add a separate PORT_MAP table to stay consistent with the existing design.